### PR TITLE
Aggregate impact on intevention output

### LINKED ISSUE #475: Impact estimates: Add expandable menu for yearly aggregates
https://github.com/stiftelsen-effekt/main-site/pull/417#issuecomment-1269498531

Image

### PR DESCRIPTION
**Pull request by fellmirr**
Closes #475 

Tested on devices 
- [x] Desktop
- [x] Mobile

Tests  
- [ ] All tests are running
- [x] Test are updated
- [x] Code Review
- [x] QA

Time spent on CR:
Time spent on QA:

### CODE REVIEW

File: components/profile/donations/DonationsDistributionTable/DonationsDistributionTable.tsx [hunk 1]
```diff
@@ -1,35 +1,182 @@
...
   const [expanded, setExpanded] = useState(true);
+  const [loadedClass, setLoadedClass] = useState<string>("");
+  const [lastimpactCount, setLastimpactCount] = useState<number>(0);
+  const [currentHeight, setCurrentHeight] = useState<number | "auto">(0);
+  const [currentTimeoutId, setCurrentTimeoutId] = useState<NodeJS.Timeout | null>(null);
+  const tableContainerRef = useRef<HTMLDivElement>(null);
+
+  const {
+    data: impactdata,
+    error: imacterror,
```
Comment (EliseAndresen) [inline components/profile/donations/DonationsDistributionTable/DonationsDistributionTable.tsx:31]: Should be impacterror?

**Review by EliseAndresen — APPROVED** [review]

**Comment by EliseAndresen** [comment]
Should nr be top aligned if name is on two rows?

Pasted Graphic

**Comment by fellmirr** [comment]
> Should nr be top aligned if name is on two rows?
> 
> Pasted Graphic

Yes, I've fixed this now :+1:

**Comment by fellmirr** [comment]
Ah, yes, it gets filtered out

**Review by EliseAndresen — APPROVED** [review]

**Pull request merged**